src/osw/data_import_utility.py

```diff
@@ -0,0 +1,716 @@
+import copy
+import re
+import uuid as uuid_module
+import warnings
+from pathlib import Path
+from typing import Any, Callable, List, Optional, Union
+
+import deepl
+import numpy as np
+from geopy import Nominatim
+from jsonpath_ng import ext as jp
+
+from osw import wiki_tools as wt
+from osw.model import entity as model
+from osw.wtsite import WtSite
+
+# Constants
+PACKAGE_ROOT_PATH = Path(__file__).parents[2]
+CREDENTIALS_FILE_PATH_DEFAULT = PACKAGE_ROOT_PATH / "examples" / "accounts.pwd.yaml"
+ENABLE_SORTING = True
+REGEX_PATTERN: dict[str, str | dict[str, str]] = {
+    "SAP OU number and name from DN": {
+        "Pattern": r"CN=(.+)([0-9]{10})-(.+),OU=Abteilungen",
```

Comment:
How FhG-specific is this?

Reply:
This could be applicable to any company that uses 10-digit SAP organizational unit numbers and German descriptions